pkg/trivyoperator/config.go

```diff
@@ -82,6 +84,30 @@ func GetDefaultConfig() ConfigData {
 	}
 }
 
+// Set sets a key on config data
+func (c ConfigData) Set(key, value string) {
```

Comment:
wonder if there is a way to support this change without adding `Set` method to `config.go` it always best to keep the config immutable (from code) after loading it

Reply:
@chen-keinan I also prefer it immutable. But I would prefer we merged this as is, and focused on refactoring this while also doing the task https://github.com/aquasecurity/trivy-operator/issues/415

Because having ConfigData immutable, but adding two keys (`KeyVulnerabilityScannerEnabled` and `KeyExposedSecretsScannerEnabled`) that are not part of the configmap, make the change more complex, and I didn't want to add them to `trivy-operator` configmap, because if we add it there will be two places to change flag (configmap and environment variable). 

Do you think releasing as is, and then I'll start on the config refactoring to fix this is okay?

Reply:
sure , Let's do it.